app/main.py

```diff
@@ -1 +1,43 @@
-#  write your code here
+class Animal:
+
+    def __init__(
+        self, name: str, appetite: int, is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> str:
+        print(f"Hello, I'm {self.name}")
```

Comment:
The method `print_name` is defined to return a string, but it actually prints a message and returns `None`. If the intention is to return the string, consider using `return` instead of `print`.